### ISSUE TITLE
Config key `cketoolbar` vs `toolbar`?

### ISSUE THREAD

**Issue by quicksketch** [backdrop-contrib/ckeditor5#113]
At some point the config key and form element for the toolbar was changed from `toolbar` to `cketoolbar`. This is unnecessary prefixing, since the settings are exclusive to the `ckeditor5` editor. It'll probably show up in core's cSpell as an invalid word too. Should we change it back?

**Comment by indigoxela** [comment]
> This is unnecessary prefixing, since the settings are exclusive to the ckeditor5 editor.

In fact, this change was essential to get things working. Reverting isn't possible without changes in core.

See also https://github.com/backdrop/backdrop-issues/issues/6203

This filter quirk not only affects CK5, but any other editor with such a setting in the filter form. If an implementation uses "toolbar", then things will break because of different variable structure for the editors.

**Comment by indigoxela** [comment]
A little off topic: I don't think, cspell should dictate the names of variables. Nice and speaking names, OK, but not being able to use (proper) variables names because of a spellchecker... :zany_face:

**Comment by quicksketch** [comment]
I'd like to put this config key back to just "toolbar", but you're right that we can't do this easily while there's the core bug. I filed a PR at https://github.com/backdrop-contrib/ckeditor5/pull/114 but it's blocked until https://github.com/backdrop/backdrop-issues/issues/6203 is fixed and we have another bug-fix release (1.26.3).

**Comment by quicksketch** [comment]
I merged the PR at https://github.com/backdrop-contrib/ckeditor5/pull/114, but @indigoxela pointed out some pretty serious flaws in the update hook as part of the core code review. I should have retested this again before merging. I filed a follow-up PR at https://github.com/backdrop-contrib/ckeditor5/pull/123 that fixes our new test coverage and corrects the update hook to save the new config values.
